binding/web/README.md

```diff
@@ -113,56 +127,30 @@ function transcriptCallback(cheetahTranscript: CheetahTranscript) {
 
 #### Initialize in Main Thread
```

Comment:
I left a comment on the rhino pr, and it applies to Leopard as well, i guess it'd better to combine the main thread section with the worker thread one as they are basically the same